gcp/pd-backed-nfs-server/README.md

```diff
@@ -1,14 +0,0 @@
-# PD Backed NFS Server
```

Comment:
I think we should supply other new README file for the new solution, what it does and for what.

Reply:
[`fc0a9b0` (#14)](https://github.com/env0/k8s-modules/pull/14/commits/fc0a9b0ea864e467bb94ee6e354c6abc1cb7f6d1)